pkg/network/common/dns.go

```diff
@@ -34,6 +34,15 @@ type dnsValue struct {
 	// DNS.GetNextQueryTime can ignore it
 	updating bool
 }
+type DNSInterface interface {
```

Comment:
blank line before that. also, maybe use `DNS` as the name of the interface, and rename the struct? (The struct type doesn't need to be exported any more now...)

Reply:
+1